cmd/zc_traverser_blob_account.go

```diff
@@ -68,7 +73,8 @@ func (t *blobAccountTraverser) traverse(processor objectProcessor, filters []obj
 			err = containerTraverser.traverse(middlemanProcessor, filters)
 
 			if err != nil {
-				return err
+				LogStdoutAndJobLog(fmt.Sprintf("failed to list blobs in container %s: %s", v.Name, err))
```

Comment:
FYI Adele, I logged a followup item in the backlog re this, with ID 5314752.  I don't suggest it should affect this PR.

Reply:
Responded on the PBI.